apps/consulting/posts/spatial-filtering-at-scale-with-dask-and-spatialpandas.md

```diff
@@ -217,13 +217,15 @@ Other approaches were considered and experimented with. AWS Athena is a serverle
 
 ## Are you working with large datasets?
 
-Reach out to Quansight for a free consultation by sending an email to [connect@quansight.com](email).
+Reach out to Quansight for a free consultation by sending an email to [connect@quansight.com][email].
```

Comment:
The website Markdown builder autodetects this as an email and formats it as a `mailto:` link without any markup at all, actually.

```suggestion
Reach out to Quansight for a free consultation by sending an email to connect@quansight.com.
```